src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+
+<InfoBanner>
+This transition to proof-of-stake will come with some changes to the way ether is supplied. Learn more about <a href="/upgrades/merge/eth-issuance/">ether issuance before and after The Merge</a>.
+</InfoBanner>
+
+## What to I need to do to get ready? {#preparing-for-the-merge}
+
+The Merge is one of the most significant and anticipated upgrades in the history of Ethereum, and although in the long-term its impact will be felt by everyone, in the near-term some folks will need to take action to be fully prepared.
+
+For everyday users, holders, or non-node-operating stakers there is **nothing you need to do**, but a few things you should be aware of heading into The Merge. [More on this below.](#users-holders)
+
+<ExpandableCard
+title="Staking node operators and providers"
+contentPreview="If you are a staker running your own node setup, or a node infrastructure provider, there are a few things you need to be aware of to be prepared for The Merge."
+id="staking-node-operators">
+Key action items include:
+
+- Running _both_ a consensus layer (CL) client as well as an execution layer (EL) client. Third-party endpoints to obtain execution data will be unavailable after The Merge.
+- Authenticating both EL and CL clients with a shared JWT secret so they can securely communicate with one another.
+
+Not completing the above items will result in your node being seen as "offline" after The Merge until both layers are synced and authenticated.
+
+- Setting a `fee recipient` address to direct your earned transaction fee tips/MEV to.
+
+Not setting a `fee recipient` will still allow your validator to behave as usual, but you will miss out on unburnt fee tips and any MEV you would have otherwise earned in blocks your validator proposes.
+
+For more detailed information and summary of links to client resources, stakers are encouraged to check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) over on the Staking Launchpad to make sure you're fully prepared for The Merge.
+
+Note for stakers using [SaaS](/staking/saas/) or [staking pools](/staking/pools/): There is nothing you need to do to prepare for The Merge. [More below on staying safe.](#users-holders)
+</ExpandableCard>
+
+<ExpandableCard
+title="Non-validating node operators and infrastructure providers"
+contentPreview="If you're operating a non-validating Ethereum node, the most significant change that comes with The Merge is the requirement to run clients for BOTH the execution layer (EL) AND the consensus layer (CL)."
+id="node-operators">
+You probably are already running an EL client, such as Geth, Erigon, Besu or Nethermind. Up until The Merge, an EL client was enough to receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" it is contained within.
```

Comment:
Commented elsewhere on this doc, but I feel strongly about us using full terms for the execution layer and consensus layer

Reply:
I agree regarding EL/CL (though less strongly agree re: PoS/PoW, so maybe I'm a hypocrite).